ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -589,6 +591,12 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal, TwoLineTitl
         titleView.avatarOverrideStyle = style
     }
 
+    /// Override BadgeLabelStyle for navigation bar
```

Comment:
Can just make the `badgeLabelStyle` property public instead of manually implementing a setter with a different name? Or maybe rename that property to `overrideBadgeLabelStyle`?

Reply:
if we do so and mark that as @objc, it doesn't allow optional values. Also, in swift we need to mark optional if want to assign nil.  So have created this approach. Please let me know your input.

Reply:
@anandrajeswaran I have created one wrapper class to achieve this, with Enum optional its not allowing to expose, Give error as attached. To enable app to set null when not needed, We need to ensure this is optional property.

<img width="1216" alt="Screenshot 2024-02-29 at 11 56 23 AM" src="https://github.com/microsoft/fluentui-apple/assets/102029000/3c8d2800-0e80-4fcc-9b06-b9f58a1e1757">

Reply:
@anandrajeswaran have updated it, removed setter method now,